src/requirement-types.d.ts

```diff
@@ -124,11 +124,27 @@ type RequirementFulfillmentStatisticsWithCoursesWithAdditionalRequirements = Req
     readonly [name: string]: RequirementFulfillmentStatisticsWithCourses;
   };
 };
+type MixedRequirementFulfillmentStatistics = {
```

Comment:
can probably use the `Readonly` type constructor here as opposed to explicitly marking every field

Reply:
This is trying to keep consistent with rest of the code